simpleWebSocket.js

```diff
@@ -0,0 +1,48 @@
+/*
+MIT License
+
+Copyright (c) 2023 Lars Grammel
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+import * as Runtime from "./detectRuntime.js";
+
+/**
+ * Creates a simplified websocket connection. This function works in both Node.js and browser.
+ */
+export async function createSimpleWebSocket(url) {
+  switch (Runtime.detectRuntime()) {
```

Comment:
Suggestion: pull out the result to a const and in the final fall-through case below (unknown runtime) log what we got.

Reply:
detectRuntime only returns the four options here or null so unfortunately we don't have more info

```
    const globalThisAny = globalThis;
  
    if (globalThisAny.EdgeRuntime) {
      return "vercel-edge";
    }
  
    if (globalThis.navigator?.userAgent === "Cloudflare-Workers") {
      return "cloudflare-workers";
    }
  
    if (globalThis.process?.release?.name === "node") {
      return "node";
    }
  
    if (globalThis.window) {
      return "browser";
    }
  
    return null;
```